Paintroid/src/main/java/org/catrobat/paintroid/ui/MainActivityNavigator.kt

```diff
@@ -207,12 +208,20 @@ class MainActivityNavigator(
     override fun showColorPickerDialog() {
         if (findFragmentByTag(COLOR_PICKER_DIALOG_TAG) == null) {
             toolReference.tool?.let {
+                val eraserColor: Int
+                if (toolReference.tool?.toolType == ToolType.ERASER) {
+                    eraserColor = Color.TRANSPARENT
+                    it.drawPaint.color = eraserColor
```

Comment:
since eraserColor is used only once you could also just assign the value directly and save a variable there